### PR TITLE
Use varying regex delimiters to reduce escaping

### PR DESCRIPTION
**Pull request by ashmaroli**
Just changing simple regexps so that you don't have to waste time reviewing..

### CODE REVIEW

File: lib/rouge/lexers/c.rb [hunk 1]
```diff
@@ -166,7 +166,7 @@ def self.builtins
         rule %r/\\([\\abfnrtv"']|x[a-fA-F0-9]{2,4}|[0-7]{1,3})/, Str::Escape
         rule %r/[^\\"\n]+/, Str
         rule %r/\\\n/, Str
-        rule %r/\\/, Str # stray backslash
+        rule %r(\\), Str # stray backslash
```
Comment (pyrmont) [inline lib/rouge/lexers/c.rb]: Is this change intentional? There are a couple more like it throughout the docs.

**Review by pyrmont — COMMENTED** [review]
Thanks for this!

File: lib/rouge/lexers/c.rb (hunk 1 above)
Comment (ashmaroli) [reply lib/rouge/lexers/c.rb]: What's wrong in this change?

Comment (ashmaroli) [reply lib/rouge/lexers/c.rb]: oh.. I see it.. The change is unnecessary...

**Review by pyrmont — APPROVED** [review]
